src/components/BatchAllocations/BatchAllocationLogs/constants.js

```diff
@@ -18,3 +18,9 @@ export const BATCH_ALLOCATION_LOG_COLUMNS = [
   BATCH_ALLOCATION_LOG_FIELDS.createdByUsername,
   BATCH_ALLOCATION_LOG_FIELDS.jobNumber,
 ];
+
+export const BATCH_ALLOCATION_LOG_STATUSES = {
```

Comment:
you could store it in `src\components\BatchAllocations\constants.js` to eliminate the potential cause of the circular dependency